app/src/main/java/jwtc/android/chess/ics/ICSClient.java

```diff
@@ -1611,6 +1636,17 @@ public void disconnect() {
 
     public void sendString(String s) {
         if (_socket == null || _socket.sendString(s + "\n") == false) {
+            switch(get_gameStartSound()){

```

Comment:
why is there a sound on sendString - it will be mostly user initiated. Am i missing something?